.github/workflows/tests.yml

```diff
@@ -37,10 +37,10 @@ jobs:
           - 8888:8000
 
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        php: [8.2, 8.3]
-        phpunit: ['10.5', '11.0.1']
+        php: [8.4]
```

Comment:
Is there any specific reason why would only support `PHP 8.4`? Instead just appending to the versioning matrix to include `[8.2, 8.3, 8.4]`.

Reply:
This is only a temporary testing solution. This is done, so that only the PHP 8.4 tests are running. These PHP versions will be added again after PHP 8.4 tests are passing.